nvflare/app_common/executors/client_api_launcher_executor.py

```diff
@@ -32,12 +32,12 @@ def __init__(
         launch_timeout: Optional[float] = None,
         task_wait_timeout: Optional[float] = None,
         last_result_transfer_timeout: float = 300.0,
-        external_pre_init_timeout: float = 60.0,
-        peer_read_timeout: Optional[float] = 60.0,
+        external_pre_init_timeout: float = 300.0,
```

Comment:
this does not directly impact the error (60 is ok), but it is fine to set it to 300 too